ReferenceError: Can't find variable: fs

I get this error:
`JavaScript console: bundle.js:6091: ReferenceError: Can't find variable: fs`

My config file is the following

``` javascript
var path = require('path');
var webpack = require("webpack");

module.exports = {
  entry: './pgapi',
  output: {
    path: path.join(__dirname, 'dist'),
    filename: 'bundle.js'
  },
  module: {
    noParse: /node_modules\/json-schema\/lib\/validate\.js/,
    loaders: [
      { test: /\.json$/, loader: 'json-loader' }
      , {
          test: /\.js$/,
          loader: "transform?brfs"
        }
    ]
  },
  resolve: {
    extensions: ['', '.webpack.js', '.web.js', '.js']
  },
  node: {
    console: true,
    fs: 'empty',
    net: 'empty',
    tls: 'empty'
  },
  plugins: [
        //new webpack.optimize.UglifyJsPlugin(),
        //new webpack.optimize.DedupePlugin(),
        new webpack.DefinePlugin({
            __DEV__: true
        })
    ]
};
```

I did some tests, the problem happens all times that `fs.readFileSync` etc. have input parameters specified as variables rather than strings:

``` shell
$ cat test.js
 var fs  = require('fs');
 var temp = fs.readFileSync('./test.json', 'utf8');
 console.log(temp);
$ node test
{ key : hello world }

$ browserify test.js -o output.js -t brfs
$ node output
{ key : hello world }

$ cat output.js 
(function e(t,n,r){function s(o,u){if(!n[o]){if(!t[o]){var a=typeof require=="function"&&require;if(!u&&a)return a(o,!0);if(i)return i(o,!0);var f=new Error("Cannot find module '"+o+"'");throw f.code="MODULE_NOT_FOUND",f}var l=n[o]={exports:{}};t[o][0].call(l.exports,function(e){var n=t[o][1][e];return s(n?n:e)},l,l.exports,e,t,n,r)}return n[o].exports}var i=typeof require=="function"&&require;for(var o=0;o<r.length;o++)s(r[o]);return s})({1:[function(require,module,exports){

 var temp = "{ key : hello world }\n";
 console.log(temp);

},{}]},{},[1]);
$ cat test2.js
 var fs  = require('fs');
 var filename='./test.json';
 var temp = fs.readFileSync(filename, 'utf8');
 console.log(temp);
$ node test2
{ key : hello world }

$ browserify test2.js -o output2.js -t brfs
$ node output2
/output2.js:4
 var temp = fs.readFileSync(filename, 'utf8');
            ^

ReferenceError: fs is not defined
    at Object.__dirname.1 (/output2.js:4:13)
    at s (/output2.js:1:316)
    at e (/output2.js:1:487)
    at Object.<anonymous> (/output2.js:1:505)
    at Module._compile (module.js:541:32)
    at Object.Module._extensions..js (module.js:550:10)
    at Module.load (module.js:458:32)
    at tryModuleLoad (module.js:417:12)
    at Function.Module._load (module.js:409:3)
    at Module.runMain (module.js:575:10)
```

wepack version is 

``` shell
$ cat node_modules/webpack/package.json | grep id
  "_id": "webpack@1.13.1",
    "webpack-dev-middleware": "^1.0.0",
```

--- a/package.json
+++ b/package.json
@@ -9,13 +9,13 @@
   "dependencies": {
     "quote-stream": "^1.0.1",
     "resolve": "^1.1.5",
-    "static-module": "^2.2.0",
+    "static-module": "^3.0.0",
     "through2": "^2.0.0"
   },
   "devDependencies": {
     "browserify": "^16.1.1",
     "concat-stream": "^1.6.0",
-    "tap": "~0.4.8",
+    "tap": "^10.7.3",
     "through": "^2.3.4"
   },
   "scripts": {

